resources/default-config.kdl

```diff
@@ -210,6 +217,10 @@ binds {
     Mod+F { maximize-column; }
     Mod+Shift+F { fullscreen-window; }
 
+    // Actions to switch layouts.
+    //Mod+Space       { switch-layout "next"; }
+    //Mod+Shift+Space { switch-layout "prev"; }
```

Comment:
```suggestion
    // Keyboard layout switching.
    Mod+Space       { switch-layout "next"; }
    Mod+Shift+Space { switch-layout "prev"; }
```
This makes sense to have by default I think.

Reply:
Not really, Most folks switch in xkb options.

Reply:
It shouldn't conflict with xkb switching right?

Reply:
They don't conflict, it's just not everyone use multiple layouts, it's normally just one.